js/skin.js

```diff
@@ -154,6 +163,8 @@ var Skin = React.createClass({
    */
 
   getDirectionParams: function(pageX, pageY) {
+    pageX = this.checkIsNumber(pageX) ? pageX : 0;
+    pageY = this.checkIsNumber(pageY) ? pageY : 0;
```

Comment:
We have a `Utils.ensureNumber` function which does this (i.e. return a default value if the input is not a number). I recommend using this instead in order to avoid code duplication.

Reply:
\+